front/lib/api/workspace.ts

```diff
@@ -176,34 +178,58 @@ export async function getMembers(
       }
     }
 
+    const user = new UserResource(UserModel, m.user!);
+
     return {
-      ...u.toJSON(),
+      ...user.toJSON(),
       workspaces: [{ ...owner, role, flags: null }],
     };
   });
 
-  return { members: usersWithWorkspaces, total };
+  return { members: usersWithWorkspaces, total, nextPageParams };
+}
+
+interface SearchMembersPaginationParams {
```

Comment:
`nit` if we want to preserve some hierarchy here we could do:
```suggestion
interface SearchMembersPaginationParams extends PaginationParams {
```

Reply:
I didn't want to do that at all :-)